src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/modal.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.4);
+$borderRadius: 4px;
+$backgroundColor: $white;
+$modalShadow: 0 0 25px 0 rgba($black, 0.25);
+
+.container {
```

Comment:
Isn't the major advantage of css-modules that we can use the same classes in multiple files without caution? `modal` is mentioned in the name of the file.